demo/gradio_demo.py

```diff
@@ -1,22 +1,25 @@
-from domain.ai_model import AiModel
 import gradio as gr
+from src.domain.ai_model import AiModel
 
 ai_model = AiModel()
-
+url = "https://www.provincia.bz.it/agricoltura-foreste/fauna-caccia-pesca/images/braunbaer_6016_L.jpg"
```

Comment:
URL = "https://www.provincia.bz.it/agricoltura-foreste/fauna-caccia-pesca/images/braunbaer_6016_L.jpg"